src/actions/applicationActions.js

```diff
@@ -42,7 +42,11 @@ export function getUserInfo() {
                 })
             })
             .catch(response => {
-                dispatch(updateApplication({userData: {invalidAccount: true}}))
+                if (response && response.response.status === 403) {
+                    dispatch(updateApplication({userData: {invalidAccount: true},
+                                                applicationData: {invalidAccount: true}}));
```

Comment:
Could you help me understand why are we setting this object for applicationData?

Because the only check I can find elsewhere is this one (and it's for the userData):
https://github.com/cern-eam/eam-light-frontend/blob/master-cern/src/Eamlight.js#L66